docs/src/main/asciidoc/writing-extensions.adoc

```diff
@@ -2535,8 +2538,9 @@ You can use the `io.quarkus.arc.runtime.BeanContainer` interface to interact wit
 <4> Runtime recorder retrieves the bean using its type.
 <5> Runtime recorder invokes the `IConfigConsumer#loadConfig(...)` method passing in the configuration objects with runtime information.
 
+[[manage-non-cdi-service]]
 ==== Manage Non-CDI Service
-A common purpose for an extension is to integrate a non-CDI aware service into the CDI based Quarkus runtime. Step 1 of this task is to load any configuration needed in a STATIC_INIT build step as we did in <<Parsing Config to Objects>>. Now we need to create an instance of the service using the configuration. Let's return to the `TestProcessor#parseServiceXmlConfig` method to see how this can be done.
+A common purpose for an extension is to integrate a non-CDI aware service into the CDI based Quarkus runtime. Step 1 of this task is to load any configuration needed in a STATIC_INIT build step as we did in <<parsing-config-to-objects>>. Now we need to create an instance of the service using the configuration. Let's return to the `TestProcessor#parseServiceXmlConfig` method to see how this can be done.
```

Comment:
```suggestion
A common purpose for an extension is to integrate a non-CDI aware service into the CDI based Quarkus runtime.
Step 1 of this task is to load any configuration needed in a STATIC_INIT build step as we did in <<parsing-config-to-objects>>.
Now we need to create an instance of the service using the configuration.
Let's return to the `TestProcessor#parseServiceXmlConfig` method to see how this can be done.
```